app/views/projects/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2><%= @project.name %></h2>
+
+<p>Theme: <%= @project.challenge.theme %></p>
+<p>Material: <%= @project.material %></p>
+<p>Number of Contestants: <%= @project.contestant_count %></p>
+<p>Average Contestant Experience: <%= @project.average_years_of_experience %> years</p>
+
+<section>
+  <h3>Add new Contestant</h3>
+
+  <%= form_tag "/projects/#{@project.id}" do %>
+    <%= label_tag :id %>
+    <%= text_field_tag :id %>
+    <%= submit_tag "Add" %>
+  <% end %>
```

Comment:
Here you go, I'd change this text_field_tag and label_tag to `:contestant_id` since that is how you have it stored in your ContestantProjects controller. 

As mentioned prior, save this post route to `/projects/:id` for when you do want the functionality of creating a project. Instead, you may consider using `/contestant_projects` and passing along query params, or just handrolling a route such as `/contestant_projects/:project_id` (since we haven't really talked much about query params this mod.